doc/index.rst

```diff
@@ -107,7 +108,7 @@ When you deploy, run ``sass:build`` command before the ``asset-map:compile`` com
     $ php bin/console sass:build
     $ php bin/console asset-map:compile
 
-Limitation: url() Relative Paths
+Limitation: ``url()`` Relative Paths
```

Comment:
@smnandre Nice catch! I still see this has not been fixed by anyone yet, could you reopen and rebase your PR? Or just create a new one, I will be happy to merge :)

Reply:
Yep! Just opened it https://github.com/SymfonyCasts/sass-bundle/pull/18

Reply:
Awesome! Thank you :)